tests/dailytest/case.go

```diff
@@ -351,34 +351,37 @@ func updatePKUK(db *sql.DB, opNum int) error {
 	maxKey := 20
 	mustExec(db, "create table pkuk(pk int primary key, uk int, v int, unique key uk(uk));")
 
-	var pks []int
+	pks := make(map[int]struct{})
+	freePks := rand.Perm(maxKey)
+
+	nextPk := func() int {
+		return freePks[0]
```

Comment:
https://godoc.org/math/rand#Shuffle before return first one or randomly pick one?

Reply:
@july2993 `rand.Perm` is enough? On deletion, the pks are deleted in random order and appended back to this slice.

Reply:
there's no way you will do thinks like this:
update pkuk set pk = 1 where pk = 2;
update pkuk set pk = 2 where pk = 1;
update pkuk set pk = 1 where pk = 2;
update pkuk set pk = 2 where pk = 1;
...
continually to swap the two values, why not just simply make it more randomly?